src/Dfe.EarlyYearsQualification.Web/libman.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The NuGet version will not get added to the wwroot folder unless it is done manually, this is just a quick and easy way of doing that instead of dragging the right files over from the downloaded files